### PR TITLE
Add navigation buttons to paginated search.

### PR DESCRIPTION
**Pull request by estacioneto**
#### What is the purpose of this pull request?

Add navigation buttons to paginated search result pages.

#### What problem is this solving?

No way to navigate through the search pages.

#### How should this be manually tested?

[Access the worspace](https://estacio--storecomponents.myvtex.com/eletronics) and test it

#### Screenshots or example usage

image

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [x] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/SearchFilter.js [hunk 1]
```diff
@@ -72,27 +77,42 @@ class SearchFilter extends Component {
...
+                    params={pagesArgs.params}
+                    query={pagesArgs.queryString}>
+                    <div className="w-90 flex items-center justify-between pa3">
+                      <div className="flex items-center justify-center">
+                        <span
+                          className="bb"
+                          style={{
+                            borderColor: `${
+                              this.isSelected(opt.Name)
+                                ? '#368DF7'
```
Comment (andreldsa) [inline react/components/SearchFilter.js]: extract to constant or use a named color.

File: react/components/SelectedFilters.js [hunk 2]
```diff
@@ -49,19 +48,33 @@ class SelectedFilters extends Component {
 
   getSelectedFilters() {
     const selecteds = this.props.selecteds
-    return reduce((a, filterName) => concat(a, map(term => {
-      return ({ key: `${filterName}:${term}`, value: term })
-    }, selecteds[filterName])), [], keys(selecteds))
+    return reduce(
+      (a, filterName) =>
```
Comment (andreldsa) [inline react/components/SelectedFilters.js]: Change `a` to a better variable.

File: react/constants/SearchHelpers.js [hunk 3]
```diff
@@ -67,6 +83,11 @@ export function getPagesArgs(opt, queryArg, mapArg, orderBy, isUnselectLink, typ
   const page = 'store/search'
   const params = { term: pathValues[0] }
   const Q = pathValues.splice(1).join(',') || undefined
-  const queryString = QueryString.stringify({ map, page: 1, O: orderBy, Q })
+  const queryString = QueryString.stringify({
+    map,
+    page: pageNumber,
+    O: orderBy,
```
Comment (andreldsa) [inline react/constants/SearchHelpers.js]: Rename to order

File: react/constants/SearchHelpers.js [hunk 4]
```diff
@@ -67,6 +83,11 @@ export function getPagesArgs(opt, queryArg, mapArg, orderBy, isUnselectLink, typ
...
+    Q,
```
Comment (andreldsa) [inline react/constants/SearchHelpers.js]: rename to query

Comment (estacioneto) [reply react/constants/SearchHelpers.js]: This is something with the backend, I think... @ClaudivanFilho did this

Comment (andreldsa) [reply react/constants/SearchHelpers.js]: 👍

File: react/components/SearchFilter.js [hunk 5]
```diff
@@ -58,9 +58,15 @@ class SearchFilter extends Component {
       title = this.props.intl.formatMessage({ id: title })
```
Comment (lucasecdb) [inline react/components/SearchFilter.js]: I know it probably wasn't you but, props aren't supposed to be changed like this, it's better to put another variable to have this condition and keep the prop unaltered.

**Pull request merged**